compiler/rustc_parse/src/parser/mod.rs

```diff
@@ -943,6 +943,10 @@ impl<'a> Parser<'a> {
                                 Err(e) => {
                                     // Parsing failed, therefore it must be something more serious
                                     // than just a missing separator.
+                                    for xx in &e.children {
+                                        // propagate the help message from sub error 'e' to main error 'expect_ee;
```

Comment:
```suggestion
                                        // propagate the help message from sub error 'e' to main error 'expect_err'
```
Is this comment supposed to be like that?

Reply:
fix was pushed. thanks for the correction